incubating.rst

```diff
@@ -111,11 +113,59 @@ Likewise, the ``home_folder``, to define an optional Conan cache location for th
       # to the workspace root folder)
       return "new" + conanws_data["home_folder"]
 
+conan workspace add/remove
+++++++++++++++++++++++++++
+
+Use these commands to add or remove editable packages to the current workspace. The ``conan workspace add <path>`` folder must contain a ``conanfile.py``.
+
+conan workspace info
+++++++++++++++++++++
+
+Use this command to show information about the current workspace
+
+.. code-block:: bash
+
+   $ cd myfolder
+   $ conan new workspace
+   $ conan workspace info
+   WARN: Workspace found
+   WARN: Workspace is a dev-only feature, exclusively for testing
+   name: myfolder
+   folder: /path/to/myfolder
+   products
+      app1
+   editables
+      liba/0.1
+         path: liba
+      libb/0.1
+         path: libb
+      app1/0.1
+         path: app1
+
+
+conan workspace open
+++++++++++++++++++++
 
 The new ``conan workspace open`` command implements a new concept. Those packages containing an ``scm`` information in the ``conandata.yml`` (with ``git.coordinates_to_conandata()``) can be automatically cloned and checkout inside the current workspace from their Conan recipe reference (including recipe revision).
 
 
-The paths in the ``conanws`` files are intended to be relative to be relocatable if necessary, or could be committed to Git in mono-repo like projects.
+conan new workspace
++++++++++++++++++++
+
+The command ``conan new`` has learned a new built-in (experimental) template ``workspace`` that creates a local project with some editable packages
+and a ``conanws.yml`` that represents it. It is useful for quick demos, proofs of concepts and experimentation.
+
+
+conan workspace build
++++++++++++++++++++++
+
+The command ``conan workspace build`` does the equivalent of ``conan build <product-path> --build=editable``, for every ``product`` defined
```

Comment:
The `limitations` section has a 
> or does any orchestrated build

notice, should that be clarified further with the new build subcommand?

Reply:
yes, lets remove it.